core/state_transition.go

```diff
@@ -385,10 +394,16 @@ func (st *StateTransition) innerTransitionDb() (*ExecutionResult, error) {
 		// After EIP-3529: refunds are capped to gasUsed / 5
 		st.refundGas(params.RefundQuotientEIP3529)
 	}
+	// effective tip - goes to coinbase
+	// base fee - goes to chaincfg sequencer
```

Comment:
Base fee goes to an address in the chain cfg, but that doesn't necessarily have to be the sequencer (it might go to the protocol in general). For now I'd not put it in comments, or it might conflict with specs / governance.